eng/common/templates/steps/init-docker-linux.yml

```diff
@@ -55,6 +55,7 @@ steps:
       -v $(Build.ArtifactStagingDirectory):$(artifactsPath)
       -w /repo
       $(dockerArmRunArgs)
+      $(imageBuilderDockerRunOptions)
```

Comment:
> As currently implemented, these are only list host run options. Should that be called out in the name or a comment?

Can you clarify what you mean here?  Not following.

Reply:
sorry I meant `linux` not list.  These are only being applied when the build is run on a Linux host.  They are not general purpose and applied to the builds run on Windows hosts.

Reply:
The name does imply that it's only for Linux, given the current implementation, because the name contains "DockerRun" and we only execute a `docker run` in the Linux scenario.  If we were to support nested Docker for Windows and execute `docker run` for the Windows scenario, then we would use this same variable for that case as well.

Reply:
Fair point - agreed.

Reply:
> This variable name implies it is the complete set of run options. Consider renaming to make this more clear.
How about `imageBuilderDockerRunExtraOptions`?

> Consider renaming dockerArmRunArgs so it aligns with this new variable. They technically could be combined but at a minimum it seems like their names should follow a similar pattern.

I'd be ok with renaming it to just something like `dockerArmRunOptions`.  But I don't think it should be beyond that in aligning with the name of the new variable because they are for different purposes.  `dockerArmRunArgs` is used for more than just calls to run ImageBuilder; it's used for Docker cleanup and testing too.

Reply:
I am fine with `dockerArmRunOptions`